templates/travel/elements/_shadow.css

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ .travel-shadow,
+.travel-shadow-hover {
```

Comment:
it should either be .travel-shadow:hover or this class should be deleted? , who adds this class in the DOM.

Reply:
Also this class is not used in this PR. Good reason to delete it